packages/create-platformatic-db/src/create-package-json.mjs

```diff
@@ -0,0 +1,33 @@
+import pupa from 'pupa'
+import { isFileAccessible } from './utils.mjs'
+import { writeFileSync } from 'node:fs'
+import { join } from 'node:path'
+
+const packageJsonTemplate = `\
+{
+  "name": "platformatic-db-api-example",
+  "version": "0.0.1",
+  "description": "Platformatic DB API",
+  "scripts": {
+    "start": "platformatic db start"
+  },
+  "devDependencies": {
+    "fastify": "^{fastifyVersion}",
+    "platformatic": "^{platVersion}"
+  },
+  "engines": {
+    "node": ">=16"
```

Comment:
The required minimum versions of Node.js for using `platformatic` are >= v16.17.0 or >= v18.8.0 ([source](https://oss.platformatic.dev/docs/getting-started/quick-start-guide#requirements)).

I think the value for this field should be adjusted to:

```
^16.17.0 || ^18.8.0
```

Worth double checking that I've got this syntax correct 😅